lib/u3d/commands_generator.rb

```diff
@@ -119,6 +119,7 @@ def run
         c.option '-o', '--operating_system STRING', String, "Checks for availability on specific OS [#{oses.join(', ')}]"
         c.option '-u', '--unity_version STRING', String, 'Checks if specified version is available. Can be a regular expression'
         c.option '-p', '--packages', 'Lists available packages as well'
+        c.option '--no_central', 'Do not use the central version cache'
```

Comment:
the option parser has a feature for dealing with on/off options

See `'--[no-]download`